containers/server-image/motd

```diff
@@ -0,0 +1,6 @@
+===============================================================================
+!
+! This shell runs in a container: not all changes will be persisted in volumes.
+! Be careful of what you modify.
+!
```

Comment:
Look ok to me. Small nitpick with message to clarify a bit more but rest looks good. 

```suggestion
! This shell operates within a container environment, meaning that not all modifications will be permanently saved in volumes.
! Please exercise caution when making changes, as some alterations may not persist beyond the current session.
```

Reply:
Suggestion taken with some reformatting to fit 80 characters line